jobtastic/tests/test_broker_fallbacks.py

```diff
@@ -11,25 +11,27 @@
     from celery.tests.case import AppCase
 # eager_tasks was removed in celery 3.1
 from jobtastic.tests.utils import eager_tasks
+from jobtastic import JobtasticTask
 
-USING_CELERY_2_X = False
 try:
     from kombu.transport.pyamqp import (
         Channel as AmqpChannel,
     )
-    from kombu.exceptions import StdChannelError, StdConnectionError
 except ImportError:
-    USING_CELERY_2_X = True
     from kombu.transport.amqplib import (
         Channel as AmqpChannel,
     )
+# Kombu>=2.1 doesn't have StdConnectionError or StdChannelError, but the 2.1
+# amqp Transport uses an IOError, so we'll just test with that.
+try:
+    from kombu.exceptions import StdChannelError
+except ImportError:
+    StdChannelError = IOError
+try:
     from kombu.exceptions import StdChannelError
```

Comment:
These are both importing `StdChannelError` but the first has a fallback for `StdChannelError` and the second for `StdConnectionError`